backend/api/views.py

```diff
@@ -153,3 +156,11 @@ def get_serializer_class(self) -> Serializer:
     def perform_create(self, serializer: Serializer) -> None:
         """Переопределение метода perform_create."""
         serializer.save(author=self.request.user)
+
+
+class FAQViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
+    "ViewSet returns a frequently asked questions list."
+    serializer_class = FAQSerializer
+
+    def get_queryset(self):
+        return cache.get_or_set(FAQ_CACHE_KEY, FAQ.objects.all, timeout=None)
```

Comment:
Будет ли оно работать без явного вызова all? Не будет ли оно бить по базе данных при каждом подобном обращении к кэшу?

Reply:
Описание функции:
    def get_or_set(self, key, default, timeout=DEFAULT_TIMEOUT, version=None):
        """
        Fetch a given key from the cache. If the key does not exist,
        add the key and set it to the default value. The default value can
        also be any callable. If timeout is given, use that timeout for the
        key; otherwise use the default cache timeout.

        Return the value of the key stored or retrieved.
        """